poetry/console/commands/init.py

```diff
@@ -158,26 +159,29 @@ def handle(self):
         )
         help_displayed = False
         if self.confirm(question, True):
-            self.line(help_message)
-            help_displayed = True
+            if self.io.is_interactive():
+                self.line(help_message)
+                help_displayed = True
             requirements = self._format_requirements(
                 self._determine_requirements(self.option("dependency"))
             )
-            self.line("")
+            if self.io.is_interactive():
+                self.line("")
```

Comment:
I'm not sure if we really need this `self.line("")`. I guess at least it can be placed directly after the last output above and we can avoid this second if-statement.

Same thing applies to line 183-184.

Reply:
It serves to separate the interactive output of `_determine_requirements`, so that would change the output. Still want the change?